### DISCUSSION TITLE
Installation attempt into an LXC container?

### DISCUSSION THREAD

**Discussion by eisengrau** [Security-Onion-Solutions/securityonion#1750] [Unsupported Versions] [answered]
Hello,

A couple of month ago I have tried, and today did another attempt to install security-onion into an LXC container with Ubuntu 20.04, 18.04 and Centos 7.

Unfortunately when I clone the repo and run the install script, sudo bash so-setup-network, nothing really happenes, the script just hangs.

I checked the script, it points to ./setup/so-setup network "$@", which also does the same... just hangs. Is there something that I missed? I checked the so-setup script itself, but there is really nothing that seems odd, at least it should give out an error prompt.

**Comment by TOoSmOotH** [comment, marked as answer]
Docker doesn't work properly in LXC containers.
